app-admin/ruri/autobuild/defines

```diff
@@ -0,0 +1,6 @@
+PKGNAME=ruri
+PKGSEC=admin
+PKGDEP="libcap libseccomp"
+PKGDES="A light linux container with zero runtime dependency"
+
+ABSTRIP=0 
```

Comment:
Why?

Reply:
https://wiki.aosc.io/zh/developer/packaging/package-styling-manual/#er-jin-zhi-bao

Reply:
> https://wiki.aosc.io/zh/developer/packaging/package-styling-manual/#er-jin-zhi-bao

You are building from source.

Reply:
> > https://wiki.aosc.io/zh/developer/packaging/package-styling-manual/#er-jin-zhi-bao
> 
> You are building from source.

So should I remove this line now?

Reply:
Quite obviously, yes.